packages/spatial/src/renderer/components/DirectionalLightComponent.ts

```diff
@@ -95,33 +98,22 @@ targetLineGeometry.setAttribute(
   new Float32BufferAttribute([-t, t, 0, 0, 0, 1, t, t, 0, 0, 0, 1, t, -t, 0, 0, 0, 1, -t, -t, 0, 0, 0, 1], 3)
 )
 
-type DirectionalLightHelper = {
-  lightPlane: LineSegments<BufferGeometry, LineBasicMaterial>
-  targetLine: LineSegments<BufferGeometry, LineBasicMaterial>
-  name: string
-  size: number
-  lightHelperEntity: Entity
-}
+const mergedGeometry = mergeBufferGeometries([targetLineGeometry, lightPlaneGeometry])
 
 export const DirectionalLightComponent = defineComponent({
   name: 'DirectionalLightComponent',
   jsonID: 'EE_directional_light',
 
   onInit: (entity) => {
-    const light = new DirectionalLight()
-    light.target.position.set(0, 0, 1)
-    light.target.name = 'light-target'
-    light.shadow.camera.near = 0.01
-    light.shadow.camera.updateProjectionMatrix()
     return {
-      light,
+      light: null! as DirectionalLight,
       color: new Color(),
       intensity: 1,
       castShadow: false,
       shadowBias: -0.00001,
       shadowRadius: 1,
       cameraFar: 200,
-      helper: null as DirectionalLightHelper | null
+      useInCSM: true
```

Comment:
merge error - useInCSM has since been removed and now exists as an inversion of control on RenderSettingsComponent.